easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -12,6 +12,7 @@ sources = [SOURCE_TAR_GZ]
 source_urls = [GNU_SOURCE]
 
 builddependencies = [
+	('M4', '1.4.17'),
```

Comment:
this has some code style issues, tab characters should not be used for indentation (see http://pep8.org/#tabs-or-spaces)